mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -6447,79 +7055,152 @@ struct GridwiseGemmV2RewritePattern
 
     // Compute ThreadSliceLengths for Matrix A.
     uint64_t GemmABlockCopyNumberDataPerThread =
-        MPerBlock * KPerBlock / BlockSize;
+        MPerBlock * KPerBlock * KPack / BlockSize;
```

Comment:
Let's put some parentheses around the multiplication to make sure we don't get messed up by `KPack / BlockSize` ending up as part of the computation (for all I know, it might)